wherehows-web/app/components/search/search-facet.ts

```diff
@@ -0,0 +1,25 @@
+import Component from '@ember/component';
+import { computed } from '@ember-decorators/object';
+import { IFacetSelections } from 'wherehows-web/utils/api/search';
+
+/**
+ * Presentation component of a facet
+ */
+export default class SearchFacet extends Component {
+  /**
+   * Facet selections
+   */
+  selections: IFacetSelections;
+
+  /**
+   * Computed property to check if there is any selection in the
+   * facet. If that is the case, a clear button will show up.
+   */
+  @computed('selections')
+  get showClear(): boolean {
```

Comment:
maybe `showClearBtn` or `showClearOption` would be a more clear name

Reply:
+1